atrope/image_list/manager.py

```diff
@@ -125,13 +126,38 @@ def _load_sources(self):
             )
 
         for name, list_meta in image_lists.items():
-            lst = atrope.image_list.hepix.HepixImageListSource(
-                name,
-                url=list_meta.pop("url", ""),
-                enabled=list_meta.pop("enabled", True),
-                subscribed_images=list_meta.pop("images", []),
-                prefix=list_meta.pop("prefix", ""),
-                project=list_meta.pop("project", ""),
-                **list_meta
-            )
-            self.lists[name] = lst
+            source_type = list_meta.pop("type", "hepix").lower()
+            if source_type == "hepix":
+                lst = atrope.image_list.hepix.HepixImageListSource(
+                    name,
+                    url=list_meta.pop("url", ""),
+                    enabled=list_meta.pop("enabled", True),
+                    subscribed_images=list_meta.pop("images", []),
+                    prefix=list_meta.pop("prefix", ""),
+                    project=list_meta.pop("project", ""),
+                    file_path=list_meta.pop("file_path", None),
+                    **list_meta
+                )
+                self.lists[name] = lst
+            elif source_type == "harbor":
```

Comment:
I think this becomes a bit confusing as the `hepix` image list file is used for things that are not `hepix`. I would change:
```
CONF.import_opt("hepix_sources", "atrope.image_list.hepix", group="sources")
```
for something more generic and we can keep the default type as `hepix` as you have in the code